blockchain/vm/jump_table.go

```diff
@@ -75,7 +75,6 @@ func newCancunInstructionSet() JumpTable {
 	enable5656(&instructionSet) // EIP-5656 MCOPY opcode
 	enable6780(&instructionSet) // EIP-6780 SELFDESTRUCT only in same transaction
 	enable1153(&instructionSet) // EIP-1153 TLOAD, TSTORE opcode
-	enable1052(&instructionSet) // EIP-1052 EXTCODEHASH fix
```

Comment:
It seems that you removed this line because EIP-1052 is never activated on Ethereum.  
However, we cannot simply remove this. This will break the backward compatibility.

One way to solve this is override `vm.CancunInstructionSet` during this test.
btw, why is Shanghai test affected by Cancun? Shanghai is older.

Reply:
Got it. Thanks.

Reply:
> It seems that you removed this line because EIP-1052 is never activated on Ethereum.
However, we cannot simply remove this. This will break the backward compatibility.

How about skipping tests related to this problem? I think it's better to test Kaia as is.

Only one test could be skipped.
- `shanghai/eip3651_warm_coinbase/warm_coinbase/warm_coinbase_gas_usage.json/tests/shanghai/eip3651_warm_coinbase/test_warm_coinbase.py::test_warm_coinbase_gas_usage[fork_Cancun-state_test-EXTCODEHASH]`

Reply:
You're right. The test should be skipped.

Reply:
FYI we patched 1052 in Kaia's Cancun version https://github.com/klaytn/klaytn/releases/tag/v1.12.0

Reply:
I add changing gas cost of opcode to enable to run the skipped test.
commit: https://github.com/kaiachain/kaia/pull/129/commits/831c67d787c6f9ecc866b99da0c462e69d36329e